hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMDbCheckpointServlet.java

```diff
@@ -70,7 +70,7 @@
   private String omId;
 
   @Rule
-  public Timeout timeout = new Timeout(60000);
+  public Timeout timeout = new Timeout(240000);
```

Comment:
Why do we need to increase timeout here?

Instead, since only 1 of the 2 tests need the MiniOzoneCluster, we can change the cluster init to @BeforeClass or pull it into the test method. We can also add .setNumDatanodes(1) since we do not need 3 DNs. On my local host, I can see a significant perf improvement. 

(Similarly for the corresponding test)

Reply:
yup, will do. perhaps my mac is slow, it takes 2-3 minutes for mini ozone cluster to come up.